rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/zk/PatchStoreZk.java

```diff
@@ -230,23 +196,11 @@ protected void initialize(LocalServerConfig config) {
         return listDataSourcesZk();
     }
 
```

Comment:
The below is no longer needed since the ZooKeeper connection is completely managed by the `ZkConnection` instance (and its collaborators, but we should consider it a black box at this level).